cpp/python/librmm_cffi/__init__.py

```diff
@@ -40,6 +40,11 @@ def _get_lib_name():
         if os.path.isfile(localpath):
             return localpath
         else:
+            lib_path = os.path.join(sys.prefix, 'lib')
```

Comment:
What does this do? (perhaps comment). Any change here will need to be made in the RMM repo.

Reply:
`sys.prefix` is where the python platform files are installed. So a system level python install, it would be `/usr/local` or in a conda environment, something like `/conda/envs/my_env/`. Joining to `lib` gets a potential path for installed shared objects.